stm32cube/stm32h7xx/drivers/include/stm32h7xx_ll_spi.h

```diff
@@ -3029,6 +3029,12 @@ __STATIC_INLINE uint32_t LL_I2S_GetStandard(SPI_TypeDef *SPIx)
 __STATIC_INLINE void LL_I2S_SetTransferMode(SPI_TypeDef *SPIx, uint32_t Standard)
 {
   MODIFY_REG(SPIx->I2SCFGR, SPI_I2SCFGR_I2SCFG, Standard);
+  
+  if(Standard == LL_I2S_MODE_SLAVE_TX || Standard == LL_I2S_MODE_MASTER_TX)
+  	MODIFY_REG(SPIx->CR1, SPI_CR1_HDDIR, SPI_CR1_HDDIR);
```

Comment:
@lbonhomme-vossloh I do prefer always having brackets surrouding code-block.

It makes easier adding logs (or whatever else), and avoids unexpected expansion on macro calls.

Reply:
I strongly agree with @smartin-vossloh, I have seen on multiple occasions wasted times due to missing curly braces where they were not mandatory. Save yourself some time and never omit the curly braces :wink:
While you change this line, you can add a space after `if` keyword.